bedrock/firefox/views.py

```diff
@@ -242,6 +242,106 @@ def send_to_device_ajax(request):
     return HttpResponseJSON(resp_data)
 
 
+def firefox_all(request, platform, channel):
+    locale = l10n_utils.get_locale(request)
+
+    # Only show the new template at /firefox/all/ until enough locales have it translated.
+    # Once we no longer need the old template we can redirect the old URLs to the unified page.
+    if platform is None and channel is None and lang_file_is_active('firefox/all-unified', locale):
```

Comment:
NIT: The `view` variable here does not contain the view, it is the response because you've called the view. I would probably just immediately return.

```suggestion
    if platform is None and channel is None and lang_file_is_active('firefox/all-unified', locale):
        return all_downloads_unified(request)
    
    return all_downloads(request, platform, channel)
```

And that's it. No real need to store the response to return after. Just a style NIT though, not a requirement.